specs/iota-rs-ENGINEERING-SPEC-0000.md

```diff
@@ -464,7 +464,7 @@ A tuple with the newly promoted `(MessageId,  Message)`.
 
 # Full node API
 
-Full node API of Bee and HORNET will still be public. Users who know these relative low level Restful API can still call them directly if they are confident and think it’s good for them. Note that both Bee and HORNET haven't finalized their APIs either. Following items and signatures might change later.
+Full node API of HORNET will still be public. Users who know this relative low level Restful API can still call it directly if they are confident and think it’s good for them. Note that HORNET hasn't finalized their APIs either. The following items and signatures might change later.
```

Comment:
Just curious, but what do we mean with finalized? I mean an API will probably always change...
But yeah, let's just get rid of the current IOTA docs asap :trollface: